### ISSUE TITLE
CANT RUN PROGRAM

### ISSUE THREAD

**Issue by Shubhan1907** [spyder-ide/spyder#7221]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "/anaconda2/lib/python2.7/site-packages/spyder/widgets/fileswitcher.py", line 397, in restore_initial_state
    for path in self.initial_cursors:
TypeError: 'NoneType' object is not iterable

## Version and main components

* Spyder Version: 3.2.6 
* Python Version: 2.7.14
* Qt Versions: 5.6.2, PyQt5 5.6 on Darwin

## Dependencies
```
pyflakes >=0.5.0  :  1.6.0 (OK)
pycodestyle >=2.3 :  2.3.1 (OK)
pygments >=2.0    :  2.2.0 (OK)
pandas >=0.13.1   :  0.22.0 (OK)
numpy >=1.7       :  1.14.0 (OK)
sphinx >=0.6.6    :  1.6.6 (OK)
rope >=0.9.4      :  0.10.7 (OK)
jedi >=0.9.0      :  0.11.1 (OK)
psutil >=0.3      :  5.4.3 (OK)
nbconvert >=4.0   :  5.3.1 (OK)
sympy >=0.7.3     :  1.1.1 (OK)
cython >=0.21     :  0.27.3 (OK)
qtconsole >=4.2.0 :  4.3.1 (OK)
IPython >=4.0;<6.0:  5.4.1 (OK)
pylint >=0.25     :  1.8.2 (OK)

```

**Comment by CAM-Gerlach** [comment]
An error with an identical traceback, #5661 , was due to opening the symbol finder, and then clicking off of it, and was apparently fixed in 3.2.5 (and I can't reproduce it by that means under Spyder 3.2.8). However, the poster is using 3.2.6, so it should have been fixed.

Therefore, please let us know more information about what specifically triggers the problem, assuming it persists after you update, or otherwise we won't be able to find and resolve it and this issue will be closed after one week (7 days). Thanks.

**Comment by CAM-Gerlach** [comment]
Closed due to lack of response. If you can provide more information, please re-open and do so. Thanks.
